resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -569,7 +569,7 @@ private[spark] object KubernetesSuite {
   val pvTestTag = Tag("persistentVolume")
   val rTestTag = Tag("r")
   val MinikubeTag = Tag("minikube")
-  val SPARK_PI_MAIN_CLASS: String = "org.apache.spark.examples.SparkPi"
+  val SPARK_PI_MAIN_CLASS: String = "org.apache.spark.examples.SparkPiWithoutAppName"
```

Comment:
Instead of affecting the existing test case, could you define `SPARK_PI_MAIN_CLASS_WITHOUT_APP_NAME` and add a new test case, please, @ulysses-you ?